device_doctor/lib/src/ios_debug_symbol_doctor.dart

```diff
@@ -109,6 +109,28 @@ class RecoverCommand extends Command<bool> {
     if (timeoutSeconds == null) {
       throw ArgumentError('Could not parse an integer from the option --timeout="${argResults!['timeout']}"');
     }
+
+    // Prompt Xcode to first setup without opening the app.
+    // This will return very quickly if there is no work to do.
+    logger.info('Running Xcode first launch...');
+    final io.ProcessResult runFirstLaunchResult = await processManager.run(<String>[
+      'xcrun',
+      'xcodebuild',
+      '-runFirstLaunch',
+    ]);
+    final String runFirstLaunchStdout = runFirstLaunchResult.stdout.trim();
+    if (runFirstLaunchStdout.isNotEmpty) {
+      logger.info('stdout from `xcodebuild -runFirstLaunch`:\n$runFirstLaunchStdout\n');
+    }
+    final String runFirstLaunchStderr = runFirstLaunchResult.stderr.trim();
+    if (runFirstLaunchStderr.isNotEmpty) {
+      logger.info('stderr from `xcodebuild -runFirstLaunch`:\n$runFirstLaunchStderr\n');
+    }
+    final int runFirstLaunchCode = runFirstLaunchResult.exitCode;
+    if (runFirstLaunchCode != 0) {
+      logger.info('Failed running `xcodebuild -runFirstLaunch` with code $runFirstLaunchCode!');
```

Comment:
@christopherfujino I don't want to throw an exception here until I see it working just in case I messed this up since I'm not sure how to test it live.  Do you think this should this be `severe` or `warning` or something else?  I bet either would move it to stderr LUCI logs which would prevent the logging from being interleaved though.

Reply:
![this_is_fine](https://user-images.githubusercontent.com/7856010/214966789-3404950e-7ead-46dd-b39a-b82b7f8d9665.jpeg)